homeassistant/components/mobile_app/webhook.py

```diff
@@ -124,17 +124,20 @@ def setup_registration(hass: HomeAssistantType, store: Store,
         return empty_okay_response(headers=headers)
 
     if webhook_type == WEBHOOK_TYPE_RENDER_TEMPLATE:
-        try:
-            tpl = template.Template(data[ATTR_TEMPLATE], hass)
-            rendered = tpl.async_render(data.get(ATTR_TEMPLATE_VARIABLES))
-            return json_response({"rendered": rendered}, headers=headers)
-        # noqa: E722 pylint: disable=broad-except
-        except (ValueError, TemplateError, Exception) as ex:
-            _LOGGER.error("Error when rendering template during mobile_app "
-                          "webhook (device name: %s): %s",
-                          registration[ATTR_DEVICE_NAME], ex)
-            return json_response(({"error": str(ex)}), status=HTTP_BAD_REQUEST,
-                                 headers=headers)
+        resp = {}
+        for key, item in data.items():
+            try:
+                tpl = item[ATTR_TEMPLATE]
+                attach(hass, tpl)
+                resp[key] = tpl.async_render(item.get(ATTR_TEMPLATE_VARIABLES))
+            # noqa: E722 pylint: disable=broad-except
+            except (ValueError, TemplateError, Exception) as ex:
```

Comment:
When does ValueError happen? When does Exception happen ? 

You're over catching exceptions. We should just catch TemplateError